package.json

```diff
@@ -44,6 +55,7 @@
     "format": "prettier --write src/**/*.{js,jsx}",
```

Comment:
Yes and no. With this, we can just remove this script. `eslint` should handle everything.

Reply:
What do you mean? `eslint` will automatically fix the files?

Reply:
Yes eslint can apply prettier updates automatically. I'll delete this script. But I think semaphore is using it. What should I do?

Reply:
Semaphore should not be using any code changing operation. This script actually changes the code, makes no sense to have it in the CI.

I say we either keep a `format` script, to automatically fix the code but using ESLint, or we remove `format` at all. In the CI, let's use the `lint` script you added only, not format.

Reply:
Yeah, we should just run `lint` and `lint-styles`. That's what we do at Aurora. Any fixes should be applied locally.

Reply:
Ping on this, can we change `format` to use ESLint instead of prettier?

Reply:
I'll remove the format script, on CI we should then run `lint` and `lint-styles`, ok? `eslint` is already configured to run `prettier`.

Reply:
Not sure that should be the solution here, since we could still use an automatic formatting command as part of development. But we can leave that for another day.